src/torrent.cpp

```diff
@@ -3258,10 +3278,8 @@ namespace {
 		protocol_version const v = r.info_hash == torrent_file().info_hashes().v1
 			? protocol_version::V1 : protocol_version::V2;
 
-		auto const interval = std::max(resp.interval, seconds32(
-			settings().get_int(settings_pack::min_announce_interval)));
```

Comment:
Ow, yes, I moved it to each tracker since it's different for websocket trackers, however I removed it while addressing the previous batch of comments somehow.